corehq/apps/app_manager/models.py

```diff
@@ -2260,6 +2260,7 @@ class CaseSearch(DocumentSchema):
     data_registry_workflow = StringProperty(exclude_if_none=True)  # one of REGISTRY_WORKFLOW_*
     additional_registry_cases = StringListProperty()               # list of xpath expressions
     title_label = LabelProperty(default={})
+    description_text = LabelProperty(default={})
```

Comment:
Naming nitpick: Why not just `description`? "description" implies that it's text, and shorter variable names are generally better.

Reply:
Also on this topic: I've grown draconian about being as consistent as possible with naming, so in this PR, I'd use the same name for the app manager model (CaseSearch) and the xml model (RemoteRequestQuery), unless there's a reason not to.

Reply:
Makes sense to me, definitely happy to switch to `description`. I think I made it two words following the example of `title_label` without thinking too hard.